testsuite/model/src/test/java/org/keycloak/testsuite/model/clientscope/ClientScopeModelTest.java

```diff
@@ -97,4 +107,73 @@ public void testClientScopes() {
             return null;
         });
     }
+
+    @Test
+    @RequireProvider(value=ClientScopeProvider.class, only="jpa")
+    public void shouldMigrateUserPropertyMapperConfigInKeycloak22ForOIDC() {
```

Comment:
I like the idea of having the tests like this, however, if we change the migration to Liquibase one, this would not work.

If we decide to have the migration the current way, we should introduce a separate migration test class so it is in one place.

Reply:
Resolved, as there is no migration any more.